.travis.yml

```diff
@@ -135,7 +135,9 @@ os: linux
 jobs:
   fast_finish: true
   allow_failures:
-  - python: nightly
+  - python:
+    - 3.7-dev
```

Comment:
it doesn't work like this. you should have separate entries in `allow_failures`

Reply:
Aha, done